spec/converse.js

```diff
@@ -353,6 +353,28 @@ describe("Converse", function() {
             expect(_converse.api.settings.get('emoji_categories')?.food).toBe(undefined);
             done();
         }));
+        
+        it("extended via settings.extend works and don't override settings passed in via converse.initialize when using a shadowRoot as root specified via converse.initialize",
```

Comment:
LMAO, can you make this a little bit shorter?

What about `it("only overrides the passed in properties"` ?

Reply:
Otherwise, looks good to be merged :)